### DISCUSSION TITLE
IRI path for relational data across multiple instances of API Platform

### DISCUSSION THREAD

**Discussion by ngrafa** [api-platform/api-platform#2179] [Q&A]
Hello,

I'm working on multiple instances of API Platform. Consider the following backend apps, both powered by API Platform and UUID as the identifiers instead of integer.

- App 1 - a single tenant architecture app that contains an entity called `Organization`.
- App 2 - a multi-tenant architecture app that contains an entity called `Tenant`.

The example below shows the `Tenant`, on App2. 

```
{
      "@id": "/tenants/0e02078b-d1c2-4e71-baf5-8f7c70ce5ba2",
      "@type": "Tenant",
      "id": "0e02078b-d1c2-4e71-baf5-8f7c70ce5ba2",
      "name": "Dummy Org",
      "organization": "/organizations/4f2fe292-a2d5-4b1d-8b7a-918c31923550",
      "country": "GB",
      "notificationLanguage": "EN-GB",
      "notificationUserAccount": "/user_accounts/f22c0e53-9e03-4104-a3aa-8927edd01b1e"
},
```

The `"organization"` attribute above currently points to a temporary `Organization` entity in App2, but we want to point to the one in App1. 

I am thinking of implementing `"organization"` as a string, and keeping the full URL prefix for an IRI identifier. 

`"organization": "https://api.app1.example.com/organizations/4f2fe292-a2d5-4b1d-8b7a-918c31923550",`

I may have answered my own question, but is this this the right approach, maybe using a DataProvider for the `Tenant` entity to App2 to fetch data from App1? 

-- Raf

**Comment by landure** [comment]
Hi,

i've just created a feature request adjacent to this question:  [#7339](https://github.com/api-platform/core/issues/7339).
